marrow/mailer/transport/ses.py

```diff
@@ -5,55 +5,55 @@
     from botocore.exceptions import ClientError
 
 except ImportError:
-    raise ImportError("You must install the boto package to deliver mail via Amazon SES.")
+    raise ImportError(
+        "You must install the boto package to deliver mail via Amazon SES.")
```

Comment:
😭 PEP-8 is intended for code entering the Python standard library, it does not apply to Marrow codebases. Because this… this is ludicrous. 120 column wrapping.

Reply:
Thanks for mentioning. I configured now to use python formatter `black` with line-length of 119. See commit.